src/iterate.js

```diff
@@ -151,6 +163,10 @@ class Iterate extends events.EventEmitter {
       // Wait until we've done all the checks to emit success
       this.emit('iteration-success', value);
       debug(`ran handler in ${diff} seconds`);
+      if (this.monitor) {
+        this.monitor.measure('iteration-time', diff * 1000);
+        this.monitor.count('successful-iteration', 1);
+      }
```

Comment:
hmm, seems off to set diff with /1000 then report with *1000.  let's do the if minTime... thing with diff * 1000 and store diff as ms and submit the real value?  I don't know if we care so much about ms accuracy, it's just weird to remove three significant digits when we don't need to.